src/services/micros/lifecycle.ts

```diff
@@ -66,13 +66,13 @@ type Callback = (data: LifecycleMessageData) => void;
 // We're interested in type "Notification" and Subject "Micros Lifecycle Notification"
 // There are also messages on this queue used internally
 interface LifecycleData {
-	Type: string;
-	Subject: string;
-	Message: string;
+	Type?: string;
+	Subject?: string;
+	Message?: string;
```

Comment:
Curious about this change, should we really make our typing looser?

Reply:
Where these are currently used it is doing a direct assignment from an any coming from parsing json or yaml.  We could keep these tighter in their definition but would need more validation when assigning objects to these interfaces

Reply:
I think we should do that IMHO, otherwise we will need to check if it is defined or not everywhere

Reply:
I have updated LifecycleData back to having them as set properties and added undefined checks after assignment.

I left LifecycleMessageData with optionals on its fields for now as it is only used in one place of the code right after being converted from json